### ISSUE TITLE
Loupedeck Live - Enable the use of the 2 vertical dial displays for labelling 

### ISSUE THREAD

**Issue by Headphaze** [bitfocus/companion#3050]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

Hi team. Great work with the Loupedeck integration. Minimal fuss getting it set up. 

I've managed to get my Loupedeck Live working with Companion, including all 26 buttons and 6 dials using a 4x8 grid. 
However, the only thing I haven't managed to do (to my best ability) is set a text and/or background colour to the 6 corresponding dial sections on the 2 vertical displays. I thought i'd be able to do this by using columns 2 and 7 of the buttons grid. 

How can make use of these two vertical displays to label the function of the dials? If it's it's not currently possible, can we at some point get this feature? 

Cheers,
Sam

<img width="675" alt="Screenshot 2024-09-24 at 21 53 44" src="https://github.com/user-attachments/assets/9be41707-29ed-4a50-addc-a8d2ff6d35e8">

**Comment by Julusian** [comment]
Its not currently possible, waiting for someone to have a plan on how it should work as simply drawing the buttons on the strip really small is very weird
See https://github.com/bitfocus/companion/issues/1311 for some light discussion

**Comment by Headphaze** [comment]
Thanks. I don't know how I missed that it was mentioned in that thread.

**Comment by cwv922** [comment]
I'm also super interested in this (enough so to make an account). Why not make it a 7x4 grid, so rows 1 and 6 can be formatted like a button? I know it can't be pressed like one, but just being able to add text would be plenty for my use.

**Comment by Julusian** [comment]
this should be in 4.1 or 4.2 (not sure which yet)
treated as 3 slimmer buttons

![Image](https://github.com/user-attachments/assets/9db6475f-dacb-48d9-8731-de937ab1ecbf)

**Comment by elliotmatson** [comment]
@Julusian I'm testing this with the Razer Stream Controller, which I thought was identical hardware to the Loupedeck Live, but I'm not seeing anything displaying on the side screens. Presses on the side screens seem to register perfectly fine though I can dig deeper later today if it would be helpful!

**Comment by Julusian** [comment]
@elliotmatson yep I forgot to include that model in this, but it should work in the next build.
